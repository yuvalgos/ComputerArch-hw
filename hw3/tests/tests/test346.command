./dflow_calc tests/test346.dat tests/test346.in p91 p5 p70 d133 p130 p12 p32 p66 p73 d96 d9 p105 d62 p33 d120 p64 p183 p160 p87 p2 d84 d105 p156 d135 p78 p188 d149 p85 p94 d123 p81 d158 d36 p55 d190 p107 d181 p130 p145 d157 p11 p10 p44 d40 p34 p104 d81 p49 p178 p137 p21 p93 d68 d191 d123 p60 d90 p30 d158 d194 d78 p33 d147 d195 d88 d137 d149 d74 p199 p187 p57 p178 p111 p164 d166 d44 p78 p91 p84 p42 d195 p137 p119 d45 p143 p73 p5 d16 d132 p127 d171 p102 p156 d161 p22 d25 p173 p85 p103 d26 p186 p179 d163 p152 p51 d146 d11 d122 p82 d122 d81 p160 d59 p164 p40 p12 p100 p4 p127 d24 d130 d168 d80 p44 d171 d159 p18 p53 d28 d85 d32 p82 p16 p177 p51 p64 p186 d49