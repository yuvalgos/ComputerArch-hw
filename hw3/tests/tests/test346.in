6 19 25 26
0 3 16 5
0 0 26 9
2 26 12 15
8 24 4 25
6 3 24 3
7 15 22 1
2 0 6 22
4 0 16 26
5 27 13 26
2 31 4 7
6 30 20 5
1 8 16 20
7 11 30 10
5 21 24 28
2 21 19 8
5 12 10 26
4 12 9 5
1 31 19 7
2 19 4 25
2 30 23 7
7 17 7 30
8 20 19 22
5 1 10 12
7 21 30 30
6 22 23 3
7 18 31 22
3 4 23 10
2 20 0 27
2 21 20 5
0 24 25 16
0 26 15 0
6 21 26 31
6 31 23 26
5 15 16 18
4 20 7 16
2 21 18 20
4 21 18 17
7 7 31 26
8 14 28 1
1 15 31 15
5 1 21 18
8 12 14 8
2 17 22 10
7 12 26 23
8 27 22 28
1 6 23 18
5 4 7 13
2 1 15 22
4 22 27 20
5 31 15 22
4 9 25 17
8 4 23 4
0 29 23 15
5 20 30 16
0 25 19 13
6 13 12 0
7 11 8 6
5 16 29 26
1 0 27 5
8 31 16 23
0 13 31 21
3 6 23 4
7 27 7 31
2 16 6 26
0 22 14 26
6 20 14 0
8 28 28 2
8 18 10 22
5 27 24 22
8 20 7 24
3 25 2 1
1 16 29 28
4 10 10 6
7 6 11 29
3 3 21 26
5 31 20 10
0 22 21 1
4 3 29 1
7 2 22 16
2 20 24 9
6 16 30 6
8 4 4 29
4 10 25 15
8 16 23 29
6 1 25 0
0 15 16 5
8 22 25 18
5 20 25 21
5 17 2 11
5 14 23 6
1 12 11 26
7 22 4 11
0 10 24 16
7 27 8 14
1 22 7 7
3 16 31 12
8 26 6 2
1 26 9 9
0 26 18 24
3 25 2 24
3 20 26 17
5 10 11 29
8 22 21 3
2 6 7 31
7 29 24 1
4 17 6 27
0 12 10 10
4 9 9 17
7 23 6 17
3 24 29 29
6 17 8 28
8 4 0 7
8 21 15 9
7 15 29 27
1 14 23 16
7 11 26 16
2 27 20 20
8 10 29 10
6 4 25 24
2 17 5 28
1 8 13 30
0 14 22 24
8 16 0 5
8 22 0 15
4 4 6 31
3 29 8 11
6 29 19 5
0 26 28 30
0 15 15 4
2 22 10 20
3 18 17 14
8 25 11 16
1 20 31 18
4 14 4 19
6 17 12 15
2 25 11 21
3 13 7 14
0 24 17 17
4 30 8 15
2 28 17 30
4 14 21 26
1 8 4 7
1 6 20 8
5 2 17 25
6 20 21 20
1 25 9 24
4 16 6 29
3 12 8 10
6 30 28 15
7 10 30 24
4 11 13 24
0 2 16 9
1 9 26 15
4 27 21 19
2 12 2 29
8 11 4 23
8 31 23 23
8 25 9 7
1 5 15 5
0 20 25 20
2 1 9 13
4 27 23 18
3 4 16 30
5 18 8 16
5 17 29 26
1 20 18 23
2 18 24 28
4 17 12 14
1 7 19 7
1 25 1 19
5 27 5 0
2 21 12 30
5 4 21 1
3 9 18 23
5 29 1 5
2 22 14 20
5 5 6 22
3 8 12 22
7 21 18 24
8 2 14 8
0 31 20 29
0 17 22 9
5 16 25 8
1 22 22 25
0 10 31 28
4 0 26 7
1 15 5 22
6 8 25 0
3 13 16 25
0 13 12 10
7 1 1 3
6 18 4 23
2 11 21 4
0 26 18 16
8 14 16 30
8 9 1 21
4 30 30 29
6 3 8 1
1 28 1 13
