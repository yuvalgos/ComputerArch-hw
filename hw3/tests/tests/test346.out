Reading the opcodes latency info from tests/test346.dat ... Got latency for 9 opcodes
Reading the program file tests/test346.in ... Found 200 instructions
getProgDepth()==253
getDepDepth(91)==98
getDepDepth(5)==23
getDepDepth(70)==57
getInstDeps(133)=={76,131}
getDepDepth(130)==163
getDepDepth(12)==0
getDepDepth(32)==21
getDepDepth(66)==83
getDepDepth(73)==100
getInstDeps(96)=={76,91}
getInstDeps(9)=={-1,3}
getDepDepth(105)==121
getInstDeps(62)=={-1,52}
getDepDepth(33)==21
getInstDeps(120)=={-1,67}
getDepDepth(64)==100
getDepDepth(183)==217
getDepDepth(160)==211
getDepDepth(87)==104
getDepDepth(2)==0
getInstDeps(84)=={-1,53}
getInstDeps(105)=={30,85}
getDepDepth(156)==179
getInstDeps(135)=={107,129}
getDepDepth(78)==97
getDepDepth(188)==181
getInstDeps(149)=={140,129}
getDepDepth(85)==104
getDepDepth(94)==130
getInstDeps(123)=={59,-1}
getDepDepth(81)==117
getInstDeps(158)=={153,38}
getInstDeps(36)=={26,35}
getDepDepth(55)==5
getInstDeps(190)=={155,185}
getDepDepth(107)==111
getInstDeps(181)=={166,175}
getDepDepth(130)==163
getDepDepth(145)==194
getInstDeps(157)=={109,109}
getDepDepth(11)==0
getDepDepth(10)==0
getDepDepth(44)==21
getInstDeps(40)=={33,34}
getDepDepth(34)==76
getDepDepth(104)==124
getInstDeps(81)=={20,74}
getDepDepth(49)==80
getDepDepth(178)==210
getDepDepth(137)==151
getDepDepth(21)==5
getDepDepth(93)==117
getInstDeps(68)=={-1,65}
getInstDeps(191)=={161,78}
getInstDeps(123)=={59,-1}
getDepDepth(60)==107
getInstDeps(90)=={-1,74}
getDepDepth(30)==0
getInstDeps(158)=={153,38}
getInstDeps(194)=={192,183}
getInstDeps(78)=={53,48}
getDepDepth(33)==21
getInstDeps(147)=={143,127}
getInstDeps(195)=={183,149}
getInstDeps(88)=={71,37}
getInstDeps(137)=={38,134}
getInstDeps(149)=={140,129}
getInstDeps(74)=={57,53}
getDepDepth(199)==197
getDepDepth(187)==223
getDepDepth(57)==79
getDepDepth(178)==210
getDepDepth(111)==23
getDepDepth(164)==173
getInstDeps(166)=={164,109}
getInstDeps(44)=={31,-1}
getDepDepth(78)==97
getDepDepth(91)==98
getDepDepth(84)==94
getDepDepth(42)==47
getInstDeps(195)=={183,149}
getDepDepth(137)==151
getDepDepth(119)==147
getInstDeps(45)=={25,-1}
getDepDepth(143)==159
getDepDepth(73)==100
getDepDepth(5)==23
getInstDeps(16)=={-1,3}
getInstDeps(132)=={116,123}
getDepDepth(127)==5
getInstDeps(171)=={159,59}
getDepDepth(102)==98
getDepDepth(156)==179
getInstDeps(161)=={153,137}
getDepDepth(22)==5
getInstDeps(25)=={-1,5}
getDepDepth(173)==214
getDepDepth(85)==104
getDepDepth(103)==108
getInstDeps(26)=={18,25}
getDepDepth(186)==60
getDepDepth(179)==197
getInstDeps(163)=={147,149}
getDepDepth(152)==173
getDepDepth(51)==62
getInstDeps(146)=={108,138}
getInstDeps(11)=={-1,-1}
getInstDeps(122)=={103,110}
getDepDepth(82)==94
getInstDeps(122)=={103,110}
getInstDeps(81)=={20,74}
getDepDepth(160)==211
getInstDeps(59)=={45,-1}
getDepDepth(164)==173
getDepDepth(40)==89
getDepDepth(12)==0
getDepDepth(100)==118
getDepDepth(4)==0
getDepDepth(127)==5
getInstDeps(24)=={20,20}
getInstDeps(130)=={118,101}
getInstDeps(168)=={155,141}
getInstDeps(80)=={30,51}
getDepDepth(44)==21
getInstDeps(171)=={159,59}
getInstDeps(159)=={129,-1}
getDepDepth(18)==17
getDepDepth(53)==92
getInstDeps(28)=={8,9}
getInstDeps(85)=={71,59}
getInstDeps(32)=={31,18}
getDepDepth(82)==94
getDepDepth(16)==5
getDepDepth(177)==210
getDepDepth(51)==62
getDepDepth(64)==100
getDepDepth(186)==60
getInstDeps(49)=={45,35}
